include/boost/geometry/extensions/triangulation/strategies/cartesian/detail/precise_math.hpp

```diff
@@ -376,6 +314,47 @@ inline RealNumber orient2d(std::array<RealNumber, 2> const& p1,
     return(D[D_nz - 1]);
 }
 
+// see page 38, Figure 21 for the calculations, notation follows the notation in the figure.
+template
+<
+    typename RealNumber,
+    std::size_t Robustness = 3
+>
+inline RealNumber orient2d(std::array<RealNumber, 2> const& p1,
+                           std::array<RealNumber, 2> const& p2,
+                           std::array<RealNumber, 2> const& p3)
+{
+    if(Robustness == 0) {
```

Comment:
@tinko92 thanks for your benchmark and immediate actions on it!
I'll just write down some minor things, including stylistic details that we're used to do differently.

This `{` should go to next line